client/my-sites/stats/site.jsx

```diff
@@ -267,23 +265,73 @@ class StatsSite extends Component {
 					</div>
 				</div>
 				<JetpackColophon />
+			</>
+		);
+	}
+
+	enableStatsModule = () => {
+		const { siteId, path } = this.props;
+		this.props.enableJetpackStatsModule( siteId, path );
+	};
+
+	renderEnableStatsModule() {
+		return (
+			<EmptyContent
+				illustration="/calypso/images/illustrations/illustration-404.svg"
+				title={ translate( 'Looking for stats?' ) }
+				line={ translate(
+					'Enable site stats to see detailed information about your traffic, likes, comments, and subscribers.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'See detailed information about your traffic, likes, comments, and subscribers.' )` *ES Score: 7*